src/site/stages/build/drupal/api.js

```diff
@@ -196,9 +196,18 @@ function getDrupalClient(buildOptions, clientOptionsArg) {
       const assembleEntityTree = entityTreeFactory(contentDir);
 
       const timerStart = process.hrtime.bigint();
-      const transformedEntities = entities.map(entity =>
-        assembleEntityTree(entity),
-      );
+      const transformedEntities = entities.map(entity => {
+        const tree = assembleEntityTree(entity);
```

Comment:
Did you mean to put this line within the try catch block? If not, how come? 🙂

Reply:
The circular reference error doesn't occur during `assembleEntityTree`. It's only when we try to call `JSON.stringify` that the error is thrown.